### ISSUE TITLE
Module registred in SpamAssasin but not adding headers when using amavis

### ISSUE THREAD

**Issue by PVasileff** [bigio/spamassassin-dmarc#9]
Hello,
thanks for DMARC plugin. I have loaded dmarc SA plugin on my AlmaLinux 8 server with ISPConfig, but I have a problem that I cant fix.

When I check the .eml file with:

`spamassassin -D < email-dmarc.eml > /root/email-dmarc.eml-debug.log 2>&1`

in results I see - plugin dmarc is loaded and works:

```
$ cat email-dmarc.eml-debug.log |grep 'dbg: check: tests'
Aug 21 12:46:07.474 [839769] dbg: check: tests=DKIM_SIGNED,DKIM_VALID,DKIM_VALID_AU,DKIM_VALID_EF,DMARC_PASS,HTML_MESSAGE,RCVD_IN_DNSWL_HI,RDNS_NONE,SPF_HELO_PASS,T_SCC_BODY_TEXT_LINE,URIBL_BLOCKED
```

but my Spamassasin not running as daemon and has called from amavisd.
When I recieve a mail - amavis does not add DMARC check in mail header:

```
X-Virus-Scanned: Cloud Home Spam System on mail.domain.com
X-Spam-Flag: NO
X-Spam-Score: -3.21
X-Spam-Level:
X-Spam-Status: No, score=-3.21 tagged_above=-999 required=5
	tests=[DKIM_SIGNED=0.1, DKIM_VALID=-0.1, DKIM_VALID_AU=-0.1,
	DKIM_VALID_EF=-0.1, HTML_MESSAGE=0.001, RCVD_IN_DNSWL_HI=-5,
	RDNS_NONE=2, SPF_HELO_PASS=-0.001, SPF_PASS=-0.001,
	T_SCC_BODY_TEXT_LINE=-0.01, URIBL_BLOCKED=0.001]
	autolearn=ham autolearn_force=no
Authentication-Results: mail.domain.com (amavisd-new);
```
As can you see all checks is same as that with spamassasin -D, but without DMARC.

I have start amavis with sa_debug = 1, to check what is wrong  but I only see:

`Aug 21 12:11:30 mail amavis[827327]: SpamAssassin loaded plugins: AskDNS, AutoLearnThreshold, Bayes, BodyEval, Check, DKIM, DNSEval, Dmarc, FreeMail, HTMLEval, HTTPSMismatch, HeaderEval, ImageInfo, MIMEEval, MIMEHeader, Pyzor, Razor2, RelayEval, ReplaceTags, SPF, SpamCop, URIDNSBL, URIDetail, URIEval, VBounce, WLBLEval, WhiteListSubject`

and I see correctly loaded Dmarc plugin.

Any ideas why tests for DMARC was not added in message header when spamassasin used with amavis ?

```
$ rpm -qa |egrep -i 'spamassassin|amavis'
amavis-2.12.1-5.el8.noarch
spamassassin-3.4.6-1.el8.x86_64
```

**Comment by bigio** [comment]
Latest version (just synced from Apache src tree) has a fix for Amavisd.
The plugin has been renamed to DMARC for consistency with the RFC.

**Comment by PVasileff** [comment]
Hello and big thanks.
I have updated files, running spamassassin -D --lint, and restart amavisd. Now everithing work fine.
Thanks again @bigio. You can close that.

**Comment by bigio** [comment]
Fixed with commit 3e3f06146a2a8cd0cbaa9741327d8cf85a46167a
